seqr/utils/elasticsearch/constants.py

```diff
@@ -426,6 +426,7 @@ def get_prediction_response_key(key):
     'prev_call': {'format_value': bool},
     'prev_overlap': {'format_value': bool},
     'new_call': {'format_value': bool},
+    'prev_num_alt': {'format_value': lambda i: None if i is None else int(i)},
```

Comment:
The type of `prev_num_alt` is an integer in the ES. Do we still need to format it?

Reply:
yes if format is omitted it defaults to string, which is wrong